app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -124,7 +125,7 @@ $include-html-global-classes: $include-html-classes;
 // $info-color: #a0d3e8;
 
 // We use these to control various global styles
-// $body-bg: $white;
+$body-bg: #222222 url(/assets/Boston_Skyline_Night_BW.png) no-repeat fixed center;
```

Comment:
Color literals like `#222222` should only be used in variable declarations; they should be referred to via variable everywhere else.